debezium-core/src/main/java/io/debezium/relational/ColumnEditorImpl.java

```diff
@@ -22,6 +22,8 @@
     private boolean optional = true;
     private boolean autoIncremented = false;
     private boolean generated = false;
+    private Object defaultValue = null;
+    private boolean isDefaultValueNull = false;
```

Comment:
Why do you need this attribute? Cannot be it be computed on-the-fly just by comparing `defaultValue == null` in `isDefaultValueNull()` method?

Reply:
If the default value in ddl is not set, the `defaultValue` is `null` and we should not set `null` at field which in schema `defaultValue`;
If the default value in ddl is set with `null`, we need to convert `null` to schema type and set schema `defaultValue` with value which is converted;

>ALTER TABLE aaatest123 ADD COLUMN ddd INT DEFAULT NULL;

The default value of column `ddd` is null in ddl, we will set `isDefaultValueNull` true and then convert null to schema type;

>ALTER TABLE aaatest123 ADD COLUMN ddd INT;

The default value of column `ddd` is not defined in ddl, we will skip this column and the field `defaultValue` in `ConnectSchema` will not be set;